COLLADA2GLTF/GLTF/GLTFMesh.cpp

```diff
@@ -83,7 +83,7 @@ namespace GLTF
             size_t attributesCount = this->getMeshAttributesCountForSemantic(semantic);
             for (size_t j = 0 ; j < attributesCount ; j++) {
                 shared_ptr <GLTF::GLTFAccessor> selectedMeshAttribute = this->getMeshAttribute(semantic, j);
-                unsigned int indexSet = j;
+                unsigned int indexSet = (int)j;
```

Comment:
I see why you left this one and a few of the others this way since it would involve changing a few method stubs as well. @tfili, what do you think? I'm fine with a few casts, but it seems like it may be wise to switch over the size_t variables to ints throughout to avoid future warnings of this nature.